.github/workflows/cintegration-s3-caller.yml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   graasp-deploy-s3-workflow:
     name: Graasp auth
-    uses: graasp/graasp-deploy/.github/workflows/cintegration-s3.yml@6fb866ebf9c7369f4c324a3ba64e079ad54f2b49
+    uses: graasp/graasp-deploy/.github/workflows/cintegration-s3.yml@5ad156f86b2de177a81546db2ddc252a2ace36ca
```

Comment:
Actually I'm wondering what this refers to? Is it the commit hash?

Reply:
It is the commit hash from the previous merge. Its safer to specify this as it points to an specific version of the reusable workflow that is used. So if there are changes on the main branch, this points to the same file.